src/sipnet/outputItems.h

```diff
@@ -3,66 +3,66 @@
 #ifndef OUTPUT_ITEMS_H
 #define OUTPUT_ITEMS_H
 
-#define OUTPUT_ITEMS_MAXNAME 64  // maximum length of output item name, including the trailing '\0'
+#include <stdio.h>
```

Comment:
This is newly added and not moved up from somewhere below I'm not seeing, correct?